README.md

```diff
@@ -1,39 +1,44 @@
-
-## Model Verification
+The RecordFlux specification language is a domain-specific language to formally specify message formats of existing real-world binary protocols. Its syntax is inspired by [Ada](https://www.adacore.com/about-ada). A detailed description of the language elements can be found in the [Language Reference](/doc/Language-Reference.md).
 
 Message specifications are automatically verified using the [Z3 theorem prover](https://github.com/Z3Prover/z3). The following invariants are proven at the specification level:
 
 * Field conditions are mutually exclusive
 * Field conditions do not contradict each other
+* Field conditions are not statically false
 * Each field is reachable on at least one path from the initial node
+* Each field has at least one path to the final node
 * Message fields are always located after the first message bit
 * Field length is never negative
 * Message fields cover all bits of a message on all paths
 * Overlaid fields are congruent with exactly one other field
 
-## Code Generation
+## SPARK Code Generation
 
-The code generator generates message parsers and generators based on message specifications. The generated parser allows to validate and dissect messages and thereby respects all specified restrictions between message fields and related messages. The generated message generator enables the creation of messages according to the message specification. By using [SPARK](https://www.adacore.com/about-spark) we are able to prove the absence of runtime errors and prevent the incorrect usage of the generated code (e.g., enforce that a field of a message is validated before accessed).
+Message parsers and generators are generated based on message specifications. The generated parser allows to validate and dissect messages and thereby respects all specified restrictions between message fields and related messages. The generated message generator enables the creation of messages according to the message specification. By using [SPARK](https://www.adacore.com/about-spark) we are able to prove the absence of runtime errors and prevent the incorrect usage of the generated code (e.g., enforce that a field of a message is validated before accessed).
 
-The code generator creates a number of packages for a specification. All basic types like integers, enumerations and arrays are collectively declared in one package. For each message a child package is generated which contains validation, accessor and setter functions for every field of the message.
+Multiple packages for a specification are generated. All basic types like integers, enumerations and arrays are collectively declared in one package. For each message a child package is generated which contains validation, accessor and setter functions for every field of the message.
```

Comment:
> Multiple packages for a specification are generated.

Better: "Multiple packages are generated for a specification"

Reply:
Fixed in be3824d.